src/app/src/main/kotlin/org/vaccineimpact/api/app/security/TokenVerifyingConfigFactory.kt

```diff
@@ -29,7 +27,7 @@ class TokenVerifyingConfigFactory(
     override fun build(vararg parameters: Any?): Config
     {
         clients.forEach {
-            it.addAuthorizationGenerator({ _, profile -> extractPermissionsFromToken(profile) })
+            it.addAuthorizationGenerator(MontaguAuthorizationGenerator(repositoryFactory))
```

Comment:
Nice, this is a lot more readable than the old approach, as well as fixing the token length issue!